managed/services/telemetry/telemetry.go

```diff
@@ -237,11 +242,34 @@ func (s *Service) prepareReport(ctx context.Context) *pmmv1.ServerMetric {
 			telemetryMetric.Metrics = append(telemetryMetric.Metrics, each...)
 		}
 	}
+
+	for _, dataSource := range dataSources {
+		err := dataSource.PostFetch(ctx)
+		if err != nil {
+			s.l.Debug("PostFetch failed", err)
```

Comment:
I think it is better to log with Error + specify where it is occuring (telemetry.id)

Reply:
Fixed.